apps/i18n/turtle/en_us.json

```diff
@@ -43,8 +43,11 @@
   "hideTurtle": "hide artist",
   "jump": "jump",
   "jumpTo": "jump to",
+  "jumpToLeftPosition": "jump to the top left position",
```

Comment:
Don't we need to change the code to use these new strings?

Reply:
I expect we also need to change this to be something more like `"jump to the {pos} position"`